packages/button/src/button.tsx

```diff
@@ -4,19 +4,32 @@ import { parsePadding, pxToPt } from "./utils";
 type ButtonElement = React.ElementRef<"a">;
 export type ButtonProps = React.ComponentPropsWithoutRef<"a">;
 
+function computeFontWidthAndSpaceCount(expectedWidth: number) {
+  let smallestSpaceCount = 0;
+
+  while (expectedWidth / smallestSpaceCount / 2 > 5) {
+    smallestSpaceCount++;
+  }
+
+  return [expectedWidth / smallestSpaceCount / 2, smallestSpaceCount];
+}
+
 export const Button = React.forwardRef<ButtonElement, Readonly<ButtonProps>>(
   ({ children, style, target = "_blank", ...props }, ref) => {
     const { pt, pr, pb, pl } = parsePadding({
       padding: style?.padding,
-      paddingLeft: style?.paddingLeft,
-      paddingRight: style?.paddingRight,
-      paddingTop: style?.paddingTop,
-      paddingBottom: style?.paddingBottom,
+      paddingLeft: style?.paddingLeft ?? style?.paddingInline,
```

Comment:
Added these because apparently we did not support padding Inline nor padding Block yet